arelle/plugin/validate/DBA/rules/fr.py

```diff
@@ -456,6 +457,37 @@ def rule_fr56(
             )
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+)
+def rule_fr75(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    DBA.FR75: DBA.FR75: The company must provide information on the number of employees.
+    The rule is activated if personnel costs (fsa:EmployeeBenefitsExpense) or salaries (fsa:WagesAndSalaries) are greater
+    than DKK 200,000, and no number of employees (fsa:AverageNumberOfEmployees) has been specified.
+    """
+    costModelFacts = []
+    employeeBenefitsExpenses = val.modelXbrl.factsByQname.get(pluginData.employeeBenefitsExpenseQn, set())
+    wagesAndSalaries = val.modelXbrl.factsByQname.get(pluginData.wagesAndSalariesQn, set())
+    employeeFacts =employeeBenefitsExpenses.union(wagesAndSalaries)
+    for fact in employeeFacts:
+        if int(fact.value) > EMPLOYEE_COST_THRESHOLD and fact.unit.value == DANISH_CURRENCY:
+            costModelFacts.append(fact)
+    if costModelFacts and isFactMissing(val.modelXbrl, pluginData.averageNumberOfEmployeesQn):
```

Comment:
Fine to merge, but we can skip a lot of work in this validation if we just check `if not isFactMissing(val.modelXbrl, pluginData.averageNumberOfEmployeesQn)` and exit first thing before we build/check `costModelFacts`.